lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -42,6 +45,14 @@ def find
         end
 
         protected
+        # @since 0.6.0
+        # @api private
+        def find_cached_template
+          partials = Lotus::View.configuration.partials[[view_template_dir, template_name].join(separator)] ||
```

Comment:
@stevehook Referencing `Lotus::View` here is unsafe. Multiple _"copies"_ of the framework, with different configuration instances are allowed within the same Ruby process.

In Lotus projects we can have `Web::View`, or `Admin::View`, with different set of partials. `Lotus::View` shouldn't have partials in its registry.

---

To solve this problem, you should reference `@view.configuration` (see `#find` in this class).